target/linux/lantiq/image/Makefile

```diff
@@ -668,84 +721,97 @@ TARGET_DEVICES += avm_fritz7360sl
 define Device/avm_fritz7412
   $(Device/AVM)
   $(Device/NAND)
+  DEVICE_VENDOR := AVM
+  DEVICE_MODEL := FRITZ!Box 7412
   BOARD_NAME := FRITZ7412
   DEVICE_DTS := FRITZ7412
   KERNEL_SIZE := 4096k
   IMAGE_SIZE := 49152k
-  DEVICE_TITLE := AVM FRITZ!Box 7412
   DEVICE_PACKAGES := kmod-ath9k kmod-owl-loader wpad-mini kmod-ltq-tapi kmod-ltq-vmmc fritz-tffs-nand fritz-caldata
 endef
 TARGET_DEVICES += avm_fritz7412
 
 define Device/avm_fritz7362sl
   $(Device/AVM)
   $(Device/NAND)
+  DEVICE_VENDOR := AVM
+  DEVICE_MODEL := FRITZ!Box 7362 SL
   KERNEL_SIZE := 4096k
   IMAGE_SIZE := 49152k
   DEVICE_DTS := FRITZ7362SL
-  DEVICE_TITLE := AVM FRITZ!Box 7362 SL
   DEVICE_PACKAGES := kmod-ath9k kmod-owl-loader wpad-basic kmod-usb-dwc2 fritz-tffs
 endef
 TARGET_DEVICES += avm_fritz7362sl
 
 define Device/arcadyan_vg3503j
+  DEVICE_VENDOR := BT Openreach
+  DEVICE_MODEL := ECI
+  DEVICE_VARIANT := VDSL Modem V-2FUb/R
   IMAGE_SIZE := 8000k
   DEVICE_DTS := VG3503J
-  DEVICE_TITLE := BT OpenReach - ECI VDSL Modem V-2FUb/R
   SUPPORTED_DEVICES += VG3503J
 endef
 TARGET_DEVICES += arcadyan_vg3503j
 
 include tp-link.mk
 
 define Device/buffalo_wbmr-300hpd
+  DEVICE_VENDOR := Buffalo
+  DEVICE_MODEL := WBMR-300HPD
   IMAGE_SIZE := 15616k
   DEVICE_DTS := WBMR300
-  DEVICE_TITLE := Buffalo WBMR-300HPD
   DEVICE_PACKAGES := kmod-mt7603 wpad-basic kmod-usb-dwc2
   SUPPORTED_DEVICES += WBMR300
 endef
 TARGET_DEVICES += buffalo_wbmr-300hpd
 
 define Device/arcadyan_vgv7510kw22-nor
+  DEVICE_VENDOR := O2
```

Comment:
So ODM is Arcadyan here -- there is no good way to squeeze this into a simple scheme of VENDOR and MODEL -- I feel a bit reminded of what iTunes does to music collections (everything has to be Artist - Album - Title)...
Anyway, we should be consistent for now and I'd be in favor of using the most low-level identifier (ie. what's printed on the board) and inventing a way to add multiple aliases for each board...